python/paddle/distributed/fs_wrapper.py

```diff
@@ -0,0 +1,223 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle.fluid as fluid
+import sys
+import abc
+import os
+from pathlib import PurePosixPath
+import shutil
+
+
+class FS(object):
```

Comment:
why not reuse framework/io.h?

Reply:
1. Need an abstract interface to hide the local and remote-fs difference.
2. Every action should return the action result and try when they meet error until timeout.
3. Many of the code under fs.h can be deleted.

Reply:
OK